vtiger_connector_calendar/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
 {
-    'name': 'Calendar VTiger Integration',
-    'version': '10.0.1.0.0',
-    'summary': 'Calendar VTiger Integration',
-    'description': """
-        Calendar VTiger Integration""",
+    'name': 'VTiger Base Connector Calender',
+    'version': '11.0.1.0.0',
+    'category': '',
```

Comment:
why category is empty ?  
Category is like Gender

Reply:
ok sir, will add it.

Reply:
**Done**